package.json

```diff
@@ -37,7 +37,7 @@
     "@edx/browserslist-config": "^1.1.1",
     "@edx/frontend-build": "12.8.51",
     "@edx/frontend-platform": "4.5.1",
-    "@edx/paragon": "^20.19.0",
+    "@edx/paragon": "21.0.0-alpha.38",
```

Comment:
Please, move to `peerDependencies`

Reply:
Done

Reply:
There is still an issue with peerDependecis since frontend-platform has peerDependecey for Paragon and doens't accept the `aplha` suffix

Reply:
Or may be not... I just had to install frontend-platform locally where I checked out to the Design tokens PR instaed of installing version `5.0.0`